lib/warnings.js

```diff
@@ -24,7 +26,15 @@ const FSTSEC001 = createWarning({
   unlimited: true
 })
 
+const FSTDEP022 = createWarning({
+  name: 'FastifyWarning',
+  code: 'FSTDPE022',
+  message: 'router options for ignoreTrailingSlash, ignoreDuplicateSlashes, maxParamLength, caseSensitive, allowUnsafeRegex, querystringParser, useSemicolonDelimiter, or constraints property access is deprecated. Please use "options.routerOptions" instead for accessing router options. The "options.ignoreTrailingSlash" will be removed in `fastify@6`.',
```

Comment:
instead of a long message, could we include a string parameter?
so we emit the right field and since this warning is `unlimited`, we will read them all (when used)

Reply:
Good recommendation, this should be fixed now